packages/serverless-components/aws-cloudfront/__tests__/general-options.test.js

```diff
@@ -130,6 +130,25 @@ describe("General options propagation", () => {
     );
   });
 
+  it("update distribution with undefined aliases does not override existing aliases", async () => {
+    // Create distribution
+    await component.default({ enabled: true, origins });
+
+    // Update distribution
+    await component.default({
+      enabled: false,
+      origins
```

Comment:
Complementing your test, you could do the following. 
```suggestion
      aliases: ["foo.example.com"],
      origins
```